gateway/src/apicast/policy/cors/schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "CORS policy configuration",
+  "type": "object",
+  "properties": {
+    "allow_headers": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "allow_methods": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": ["GET", "HEAD", "POST", "PUT", "DELETE", "PATCH"]
```

Comment:
To be future-proff we definitely should include `OPTIONS` and when we are at it I think we can include `TRACE` and `CONNECT` too. https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods

Reply:
Right.